wgsl/index.bs

```diff
@@ -2660,6 +2708,76 @@ TODO: *Stub*: Expression evaluation
 
 ### Uniformity restrictions TODO ### {#uniformity-restrictions}
 
+## Compute Shaders and Workgroups TODO ## {#compute-shader-workgroups}
+
+A <dfn noexport>workgroup</dfn> is a set of invocations which
+concurrently execute a [=compute shader stage=] [=entry point=],
+and share access to shader variables in the `workgroup` storage class.
+
+The <dfn noexport>workgroup grid</dfn> for a compute shader is the set of points
+with integer coordinates *(i,j,k)* with:
+
+*  0 &leq; i &lt; workgroup_size_x
+*  0 &leq; j &lt; workgroup_size_y
+*  0 &leq; k &lt; workgroup_size_z
+
+where *(workgroup_size_x, workgroup_size_y, workgroup_size_z)* is
+the value specified for the [=workgroup_size=] attribute of the
+entry point, or (1,1,1) if the entry point has no such attribute.
+
+There is exactly one invocation in a workgroup for each point in the workgroup grid.
+
+A compute shader begins execution when a WebGPU implementation
+removes a `GPUComputePassEncoder`'s `dispatch` or `dispatchIndirect` command
+from a queue and begins the specified work on the GPU.
+The <dfn noexport>dispatch size</dfn>,
+provided directly as arguments to the `dispatch` command, or indirectly
```

Comment:
This sounds quite like a piece that WGSL doesn't need to be concerned about. Can it say that WebGPU host API provides the dispatch size and not go into specifics?

Reply:
Sounds good to me.  I needed a way to introduce the dispatch-size as a triple of integers.